content/yaml/yaml-getting-started.md

```diff
@@ -463,3 +466,16 @@ workflows:
 Working directory paths are relative to the repository clone directory, e.g. if `mobile` is the working directory, then the script will be executed in `/Users/builder/clone/mobile`.
 
 Note that you can specify an absolute path as a working directory as well.
+
+### Labels
+
+You may use `codemagic.yaml` to define several labels for your builds. Labels also support environment variables as shown in the snippet below.
```

Comment:
What's missing for me here is information about why is this useful and what can I do with them after I define them